src/frontend/view/view.ts

```diff
@@ -121,52 +127,59 @@ const viewMainLayout = () => {
   return result.join('')
 }
 
+const viewDataBaseError = () => {
+  const html = []
+  let errorText = ''
+  switch (state.systemErrorText) {
+    case 'TIMEOUT':
+      errorText =
+        'The databse took too long to respond. This is usually due to too many query results. Please try a more precise query.'
```

Comment:
```suggestion
        'The database took too long to respond. This is usually due to too many query results. Please try a more precise query.'
```

Reply:
Are timeouts always database errors? What if the server times out for another reason?

Reply:
not sure, maybe one for @nacnudus